src/mpid/ch4/generic/am/mpidig_am_recv.h

```diff
@@ -109,7 +109,8 @@ MPL_STATIC_INLINE_PREFIX int MPIDIG_handle_unexpected(void *buf, MPI_Aint count,
     rreq->status.MPI_SOURCE = MPIDIG_REQUEST(rreq, rank);
     rreq->status.MPI_TAG = MPIDIG_REQUEST(rreq, tag);
 
-    MPIDIG_copy_from_unexp_req(rreq, buf, datatype, count);
+    mpi_errno = MPIDIG_copy_from_unexp_req(rreq, buf, datatype, count);
+    MPIR_ERR_CHECK(mpi_errno);
```

Comment:
I am on the 2nd commit. This change (adding return) belong to the previous commit.

Reply:
Fixed.